pywbemcli/_common.py

```diff
@@ -449,9 +473,10 @@ def split_array_value(string, delimiter):
 
 
 def split_value_str(string, delimiter):
-    """Spit a string based on a delimiter character bypassing escaped
-       instances of the delimiter.  This is a generator function in that
-       it yields each time it separates a value.
+    """
```

Comment:
Typo "Spit".
Indentation of the first paragraph should be one more.
Indentation of the last paragraph should now also be adjusted to match the first.

Reply:
DONE